src/content/docs/mlops/integrations/openai-integration.mdx

```diff
@@ -75,24 +82,144 @@ openai.Completion.create(
+  >
+<Steps>
+<Step>
+### Install OpenAI [#install-openai-node]
+
+Use your favorite package manager and run:
+
+```
+npm install @newrelic/openai-observability openai
+```
+</Step>
+<Step>
+### Getting started [#getting-started-node]
+
+Call `monitorOpenAI` with the `openAI`instance. 
+
+Here is some example code:
+
+```
+import { Configuration, OpenAIApi } from 'openai';
+import { monitorOpenAI } from '@newrelic/openai-observability';
+
+const configuration = new Configuration({
+  apiKey: 'OPENAI_API_KEY',
+});
+
+const openAIApi = new OpenAIApi(configuration);
+
+monitorOpenAI(openAIApi, {
+  applicationName: 'MyApp',
+  newRelicApiKey: 'NEW_RELIC_LICENSE_KEY',
+});
+
+const response = await openAIApi.createChatCompletion({
+  model: 'gpt-3.5-turbo',
+  messages: [{ role: 'user', content: 'What is Observability?' }],
+});
+
+console.log(response.data.choices[0].message);
+```
+
+Make sure that you add module type to package.json:
+
+```
+"type": "module"
+```
+</Step>
+
+<Step>
+### Initialization options [#modify-code]
+
+```
+export interface MonitorOpenAIOptions {
+  /**
+   * Your application name in New Relic, must provided.
+   */
+  applicationName: string;
+  /**
+   * API key with insert access used to authenticate the request.
+   * For more information on creating keys, please see:
+   * https://docs.newrelic.com/docs/insights/insights-data-sources/custom-data/introduction-event-api#register
+   */
+  newRelicApiKey?: string;
+  /**
+   * Optional host override for event endpoint.
+   */
+  host?: string;
+  /**
+   * Optional port override for trace endpoint.
+   */
+  port?: number;
+}
+```
+</Step>
+<Step>
+### Environment variables [environment-variables-node]
 
-## EU account users [#EU-account-users]
+You can use your configured environment variable for initialization options:
+
+`NEW_RELIC_LICENSE_KEY - API`: Key with insert access used to authenticate the request.
+
+`NEW_RELIC_INSERT_KEY`: Same as API key.
+
+`EVENT_CLIENT_HOST`:  Optional host override for events endpoint.
+</Step>
+<Step>
+### Test 
+
+If you are contributing to the package, make sure the tests are passing or update them is needed. To run the tests:
+
+```
+$ npm run test
+```
+It's important to note that  `monitor.integration` test requires `.env.test` file with `OPENAI_API_KEY` configured.
+</Step>
+</Steps>
+
+</Collapser>
+
+<Collapser 
+    id="eu-account-users" 
+    title="EU account users"
+  >
 
 Are you reporting data to the New Relic EU region? Click [here](#EU-account-users) for more instructions. If you're using an EU account, you should also set your `EVENT_CLIENT_HOST`:
 
+```bash
+export EVENT_CLIENT_HOST="insights-collector.eu01.nr-data.net"
+```
+
 ```python
 import os
 os.environ["EVENT_CLIENT_HOST"] = "insights-collector.eu01.nr-data.net"
 ```
 
 `EVENT_CLIENT_HOST` can also be sent as a parameter at the `monitor.initialization()` call.
 
+</Collapser>
 
-## Demonstration video [#demo-walthrough]
+<Collapser 
```

Comment:
I'm not convinced that this video should be in a collapser. I think if you include a video at the end of the doc, letting customers know to jump down to a different section would be helpful

Reply:
Done